modules/fts/pages/fts-introduction.adoc

```diff
@@ -0,0 +1,124 @@
+
+= Introduction to Full Text Search
+
+== Full Text
+
+Provided by the xref:services-and-indexes/services/search-service.adoc[Search Service], this is a specially purposed index, which contains targets derived from the textual contents of documents within one or more specified keyspaces.
+
+Text-matches of different degrees of exactitude can be searched for.
+Both input and target text-values can be purged of irrelevant characters (such as punctuation marks or html tags).
```

Comment:
I understand that we can ignore punctuation when building the index, but what does it mean that we can "purge them from the input and target"?

Reply:
Update: Delete line 9